### DISCUSSION TITLE
How can I exclusively return just the status?

### DISCUSSION THREAD

**Discussion by ilshm** [honojs/hono#2631] [Q&A] [answered]
How can I achieve the equivalent of Express's `sendStatus()` method in terms of returning only the status code? When I try to return `c.status(200)`, it throws an error. Thanks.

**Comment by NicoPlyley** [comment, marked as answer]
I don't believe there is a way like express has. I assume express is just setting the body to plain text. To achieve the same result you can do the following:

```ts
return c.text('', 200);
```
